src/libraries/icubmod/rawValuesPublisherClient/RawValuesPublisherClient_params.md

```diff
@@ -0,0 +1,5 @@
+* |   |  remote                | string  | -       |   -           | Yes          | Prefix of the ports to which to connect, opened by RawValuesParameterSensorsServer device. |       |
+* |   |  local                 | string  | -       |   -           | Yes          | Port prefix of the ports opened by this device. |       |
+* |   |  timeout               | double  | s       | 0.02          | No           | Timeout after which the device reports an error if no measurement was received.  |       |
+* |   |  externalConnection    | bool    | -       | false         | No           | If set to true, the connection to the rpc port of the RVP server is skipped and it is possible to connect to the data source externally after being opened | Use case: e.g yarpdataplayer source. Note that with this configuration some information like sensor name, frame name and sensor number will be not available.|
```

Comment:
Maybe the last sentence about the frame name is wrong in this context... I guess it is a copy/paste error ;)